tests/basilisp/compiler_test.py

```diff
@@ -5405,6 +5405,7 @@ def test_aliased_namespace_not_hidden_by_python_module(
         self, lcompile: CompileFn, monkeypatch: MonkeyPatch
     ):
         with TemporaryDirectory() as tmpdir:
+            cwd = os.getcwd()
```

Comment:
Is this one still needed?

Reply:
sorry I had to revert  the relevant change in the previous commit, this is needed as discussed in the similar change, because monkeypatch holds a lock to the directory while the `with TemporaryDirectory()` clause is trying to delete it on when it goes out of scope.